src/conf_macros.m4

```diff
@@ -23,12 +23,8 @@ CONFIGURABLE_VALUE(config-dir, config_dir, AUTHSELECT_CONFIG_DIR, DIR,
                    $sysconfdir/authselect)
 
 CONFIGURABLE_VALUE(profile-dir, profile_dir, AUTHSELECT_PROFILE_DIR, DIR,
-                   [Path to the directory where are stored authselect default profiles],
-                   $datarootdir/authselect/default)
-
-CONFIGURABLE_VALUE(vendor-dir, vendor_dir, AUTHSELECT_VENDOR_DIR, DIR,
-                   [Path to the directory where are stored profiles created by 3rd-party vendors],
-                   $datarootdir/authselect/vendor)
+                   [Path to the directory where are stored authselect default and vendor profiles],
```

Comment:
A bit better: Path to the directory where authselect default and vendor profiles are stored